thumbnails/thumbnails.js

```diff
@@ -1106,6 +1106,7 @@ var CarouselScroller = function(carousel, settings) {
             this.carousel.isSpringing = false;
             var pageNumber = parseInt($.data(event.target, "page"));
             yudu_commonFunctions.goToPage(pageNumber);
+            yudu_events.emit(yudu_events.ALL, yudu_events.THUMBNAILS.HIDE_THUMBNAILS_PAGE_NOT_CHANGED);
```

Comment:
@Ryan-Yudu The event is `HIDE_THUMBNAILS_PAGE_NOT_CHANGED` but it looks like it's being fired here in a situation where the page _is_ being changed?

Reply:
Good spot. This function calls `goToPage()` but in the case where you are already on that page, in the `toggle()` function, there is a branch where it does nothing but hide the thumbnails